src/components/countries/country.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { BiChevronRightCircle } from 'react-icons/bi';
+import countryName from './countries_name';
+import './scss/country.scss';
+
+const Country = ({
+  country, totalConfirmed, handleClick,
+}) => {
+  const navigate = useNavigate();
+  const HandleRoute = (route) => {
+    navigate(route);
+  };
+  const isoName = countryName[country];
+
+  const mapImg = isoName ? `https://raw.githubusercontent.com/djaiss/mapsicon/33ba28808f8d32b5bae0ffada9cadd07073852e1/all/${isoName.toLowerCase()}/vector.svg` : '/World_map.png';
+  return (
+    <>
+      <button
+        className="country"
+        type="button"
+        onClick={() => {
+          HandleRoute('/country');
+          handleClick(country);
+        }}
+      >
+        <BiChevronRightCircle size={20} className="info-icon" color="white" />
+        <img className="country-map" alt="map" src={mapImg} width="100" height="100" />
```

Comment:
- [ ] The country image in the picture below is broken, please let's fix it so that your app looks more professional.

<img width="70" alt="Loom_ujLQb31mM9" src="https://user-images.githubusercontent.com/81468675/153694295-9a27a522-32fe-4450-a97b-e73e43a579c4.png">